plugins/modules/redfish_storage_volume.py

```diff
@@ -69,6 +69,7 @@
         information.
       - >-
         C(StripedWithParity) The volume is a device which uses parity to retain redundant information.
```

Comment:
Reorder the content as below:

NonRedundant
Mirrored
StripedWithParity
SpannedMirrors
SpannedStripesWithParity